homework-submission/week1-h.w/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import "./App.css";
+import Item from "./Item.js";
+import ItemData from "./ItemData.js";
+import Header from "./Header";
+
+class App extends Component {
+  render() {
+    const Data = ItemData.map(Data => (
+      <Item text={Data.text} deadline={Data.deadline} />
```

Comment:
if you do like:
ItemData.map((Data, key) => (

                  < Item 
                      text={Data.text} 
                      deadline={Data.deadline}
                      key={key}
                    >
                  </Description>)

               ))
then you will pass the error in the chrome inspector